packages/core/src/extensions/KeyboardShortcuts/KeyboardShortcutsExtension.ts

```diff
@@ -543,6 +543,7 @@ export const KeyboardShortcutsExtension = Extension.create<{
       Backspace: handleBackspace,
       Delete: handleDelete,
       Enter: handleEnter,
+      "Shift-Enter": () => this.editor.commands.setHardBreak(),
```

Comment:
Unless something is using the `editor.commands.setHardBreak()`, I'd agree with @YousefED that this can probably even just be inlined into here & remove the `setHardBreak` command.